sdks/python/apache_beam/typehints/typed_pipeline_test.py

```diff
@@ -98,6 +98,38 @@ def process(self, element):
       [1, 2, 3] | (beam.ParDo(my_do_fn) | 'again' >> beam.ParDo(my_do_fn))
 
 
+# NativeTypesTest is only run if the typing module is available.
```

Comment:
<!--new_thread; commit:86f15f403df6cb7ad3e9d563c56d82cc4c4ac96a; resolved:0-->
Please remove conditional import, as we will depend directly on `typing`.